FrontEnd/src/components/ProfilePage/FormComponents/GeneralInfo.js

```diff
@@ -90,13 +64,17 @@ const ERRORS = {
 const TEXT_AREA_MAX_LENGTH = 1000;
 const IMAGE_SIZE = 50 * 1024 * 1024;
 
+const fetcher = (...args) => fetch(...args).then(res => res.json());
+
 const GeneralInfo = (props) => {
     const [user, setUser] = useState(props.user);
     const [formStateErr, setFormStateErr] = useState(ERRORS);
     const [imageBannerError, setImageBannerError] = useState(null);
     const [imageLogoError, setImageLogoError] = useState(null);
     const [edrpouError, setEdrpouError] = useState(null);
 
+    const { data: fetchedRegions, error, isLoading } = useSWR('http://127.0.0.1:8000/api/regions/', fetcher);
```

Comment:
Could we use `.env` for storing the base URLs?

Reply:
Our team is currently working on this, too We`ll implement it)